content/about/notes/1.1/index.md

```diff
@@ -18,15 +18,50 @@ TODO: Need to have an explicit section on everything that's being deprecated.
 
 ## Upgrades
 
-- **Helm Changes**.
-TBD: need full content & links
-
-- things to watch out for/configure before you upgrade - from all subsystems below. For example, outbound traffic is allowed by default for unknown ports, what else?
-TBD: need full content & links
-
-    - Egress gateway is disabled by default
-    - Mixer policy is off by default
-    - Outbound traffic policy is set to `ALLOW_ANY`. Traffic to unknown ports will be forwarded as-is.
+- **New `istio-init` Helm Chart**. Istio’s CRDs have been placed into their own Helm chart `istio-init`.
+  This preserves custom resource data when using `helm install`, facilitates the upgrade process,
+  and enables the Istio project to evolve beyond a Helm-based installation.
+- **Improved upgrade approach**.  The [upgrade procedure](/docs/setup/kubernetes/upgrade/) provides
+  the proper procedure for upgrading from Istio 1.0.6 to Istio 1.1.0.  If upgrading, please follow instructions
+  **carefully and precisely**.  If using Istio's built-in certmanager implementation, you must use the
+  `--set certmanager=true` flag when installing both istio-init and istio charts with either
+  `template` or `tiller` installation modes.
+- **Improved Multicluster Integration**. The 1.0 `istio-remote` chart used for
+  [multicluster VPN](/docs/setup/kubernetes/install/multicluster/vpn/) and
+  [multicluster split horizon](/docs/examples/multicluster/split-horizon-eds/) remote cluster installation
+  has been consolidated into the Istio chart.  This simplies the operator experience. To generate a 1.0
+  equivalent `istio-remote` chart, use the flag `--set global.istioRemote=true`.
+- **Reduced load balancer requirements for addons**.  Addons are no longer exposed via separate load balancers.
+  Instead addons are exposed via the ingressgateway.  To expose an addon via the ingressgateway,
+  please use the [Addon Gateway documentation](/docs/tasks/telemetry/gateways/).
+  -**More flexibility with statsd collector**.  The built-in Istio statsd collector has been removed.
+  Istio retains the capability of integrating with your own statsd collector for improved flexibility with existing
+  Kubernetes deployments.
+- **Secure addon credentials**.  Grafana, Prometheus, Kiali, and Jaeger passwords and username are now stored in
+  [Kubernetes secrets](https://kubernetes.io/docs/concepts/configuration/secret/) instead of command line configuration
+  options, `values.yaml`, or configmaps for improved security and compliance.
+- **Jaeger Introduction**. Jaeger replaces zipkin as the default tracing system.
```

Comment:
I swear this was a 1.0 thing.

Reply:
not clear.  From the change logs, jaeger was available in 1.0, but it had not replaced zipkin in 1.0.  Perhaps this section just needs changing - or deletion since zipkin can be addressed in other parts of the release notes (telemetry).